src/modules/reporting/actions/get-payout-numerators.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
This is actually a selector, not an action since it’s not making any calls or updating state.

Move this to the selectors directory + also add memoization.